plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestObjectIdFunctions.java

```diff
@@ -42,6 +45,24 @@ public void testObjectidTimestamp()
                 toTimestampWithTimeZone(ZonedDateTime.of(1979, 9, 5, 22, 51, 36, 0, UTC)));
     }
 
+    @Test
+    public void testObjectid()
+    {
+        assertFunction(
+                "ObjectId('12 34 56 78 90 ab cd ef   12 34 56 78')",
+                OBJECT_ID,
+                new SqlVarbinary(new ObjectId("1234567890abcdef12345678").toByteArray()));
+    }
+
+    @Test
+    public void testTimestampObjectid()
+    {
+        assertFunction(
+                "timestamp_objectid((TIMESTAMP '1979-09-05 22:51:36 +00:00'))",
```

Comment:
Please remove a redundant parentheses and add another case with `null`.

Reply:
Removed redundant parentheses and added a test case with `null`.